assets/global.js

```diff
@@ -879,7 +873,45 @@ class SlideshowComponent extends SliderComponent {
   }
 }
 
-customElements.define('slideshow-component', SlideshowComponent);
+customElements.define('slideshow-section-component', SlideshowSectionComponent);
+
+class AnnouncementBarSlider extends SlideshowSharedComponent {
+  constructor() {
+    debugger;
+    super();
+    this.announcementBarArrowButtonWasClicked = false;
+    this.desktopLayout = window.matchMedia('(min-width: 750px)');
+    this.reducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)');
+
+    [this.reducedMotion, this.desktopLayout].forEach((mediaQuery) => {
+      mediaQuery.addEventListener('change', () => {
+        if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
+      });
+
+      if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay()
```

Comment:
This is the only code that I repeat in both components although it's the same. But I could't find a way to keep it in the parent element. I think it's okay.